src/screens/ChannelListScreen/elements/components/PostNotificationPreview.js

```diff
@@ -111,7 +111,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     flex: 1,
     borderWidth: 1,
-    borderColor: COLORS.white
+    borderColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the `borderColor` from `COLORS.white` to `COLORS.almostBlack` could impact visibility of the border depending on the background color. Ensure that this change doesn't negatively affect the UI.
```diff
-    borderColor: COLORS.white
+    borderColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->